examples/prometheus/prometheus.yaml

```diff
@@ -205,37 +309,46 @@ objects:
             name: web
           volumeMounts:
           - mountPath: /etc/alertmanager
-            name: alertmanager-config
+            name: prometheus-alertmanager-config
           - mountPath: /alertmanager
-            name: alertmanager-data
+            name: prometheus-alertmanager-data
 
         restartPolicy: Always
         volumes:
+        #prometheus mounts
         - name: prometheus-config
           configMap:
             defaultMode: 420
             name: prometheus
-        - name: prometheus-secrets
+        - name: prometheus-proxy-secret
           secret:
             secretName: prometheus-proxy
-        - name: prometheus-tls
+        - name: prometheus-tls-secret
           secret:
             secretName: prometheus-tls
         - name: prometheus-data
           emptyDir: {}
-        - name: alertmanager-config
+        #alertmanager mounts
+        - name: prometheus-alertmanager-config
           configMap:
             defaultMode: 420
-            name: prometheus-alerts
-        - name: alerts-secrets
+            name: prometheus-alertmanager
+        - name: prometheus-alertmanager-tls-secret
           secret:
-            secretName: alerts-proxy
-        - name: alerts-tls
+            secretName: prometheus-alertmanager-tls  
+        - name: prometheus-alertmanager-proxy-secret
```

Comment:
Remove the prometheus- prefix on these - `prometheus-alertmanager-proxy` is redundant, `alertmanager-proxy` is sufficient